### ISSUE TITLE
Grade: Needs Improvement

### ISSUE THREAD

**Issue by Rodrigap** [erinl3/prj-rev-bwfs-dasmoto#5]
Good job with this! Your project looks great and your CSS especially was awesome, but I'm giving you this grade just because of the little issues with your HTML. You used all the appropriate tags, which is really important, but remember to close all your tags, be consistent with indentations, and look into properly linking your CSS file. Fixing these small issues would go a long way in making your code look really professional! Lastly feel free to change up some more of the colors and fonts in projects like these, or add your own styling elements!

**Comment by Lehla3** [comment]
Thanks for all of the feedback; I'll be sure to incorporate it all as I progress! Coding has been a lot of fun so far but there are many rules to remember at once!

So, to be clear, if I turn in a future project I'm able to get more creative with it than dictated? I want to be sure I know when to do just what we're told and when we can run free. Thanks again!

**Comment by Rodrigap** [comment]
Yeah, absolutely! Most of the project specs outline the requirements, but you're always free to elaborate on any parts or add features! You should feel free to change colors and add headers, etc.
